packages/components/src/box-control/stories/index.story.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import BoxControl from '../';
+import type { BoxControlValue } from '../types';
+
+const meta: Meta< typeof BoxControl > = {
+	title: 'Components (Experimental)/BoxControl',
+	component: BoxControl,
+	argTypes: {
+		onChange: { action: 'onChange' },
+	},
+	parameters: {
+		controls: { expanded: true },
+		docs: { canvas: { sourceState: 'shown' } },
+	},
+};
+export default meta;
+
+const defaultSideValues = {
+	top: '10px',
+	right: '10px',
+	bottom: '10px',
+	left: '10px',
+};
+
+const Template: StoryFn< typeof BoxControl > = ( props ) => {
+	const [ values, setValues ] =
+		useState< BoxControlValue >( defaultSideValues );
+
+	return (
+		<BoxControl
+			label="Padding"
+			values={ values }
+			{ ...props }
+			onChange={ ( nextValue ) => setValues( nextValue ) }
+		/>
+	);
+};
+
+export const Default = Template.bind( {} );
+Default.args = {
+	label: 'Box Control',
+	values: undefined,
```

Comment:
I understand that this aims at replicating how the Storybook examples are currently on `trunk`, but I'm not sure this is the most elegant way. We could either:

- remove this line, and always use the component in controlled mode (and with a default value)
- alternatively, we could prepare another uncontrolled template, and add one extra storybook example to make the difference clear

<details>
<summary>The second option may look this like</summary>

```diff
diff --git a/packages/components/src/box-control/stories/index.story.tsx b/packages/components/src/box-control/stories/index.story.tsx
index 3186ba23c5..9ef49f1157 100644
--- a/packages/components/src/box-control/stories/index.story.tsx
+++ b/packages/components/src/box-control/stories/index.story.tsx
@@ -34,7 +34,11 @@ const defaultSideValues = {
 	left: '10px',
 };
 
-const Template: StoryFn< typeof BoxControl > = ( props ) => {
+const TemplateUncontrolled: StoryFn< typeof BoxControl > = ( props ) => {
+	return <BoxControl { ...props } />;
+};
+
+const TemplateControlled: StoryFn< typeof BoxControl > = ( props ) => {
 	const [ values, setValues ] =
 		useState< BoxControlValue >( defaultSideValues );
 
@@ -48,28 +52,32 @@ const Template: StoryFn< typeof BoxControl > = ( props ) => {
 	);
 };
 
-export const Default = Template.bind( {} );
+export const Default = TemplateUncontrolled.bind( {} );
+Default.args = {
+	label: 'Box Control',
+};
+
+export const Controlled = TemplateControlled.bind( {} );
 Default.args = {
 	label: 'Box Control',
-	values: undefined,
 };
 
-export const ArbitrarySides = Template.bind( {} );
+export const ArbitrarySides = TemplateControlled.bind( {} );
 ArbitrarySides.args = {
 	sides: [ 'top', 'bottom' ],
 };
 
-export const SingleSide = Template.bind( {} );
+export const SingleSide = TemplateControlled.bind( {} );
 SingleSide.args = {
 	sides: [ 'bottom' ],
 };
 
-export const AxialControls = Template.bind( {} );
+export const AxialControls = TemplateControlled.bind( {} );
 AxialControls.args = {
 	splitOnAxis: true,
 };
 
-export const AxialControlsWithSingleSide = Template.bind( {} );
+export const AxialControlsWithSingleSide = TemplateControlled.bind( {} );
 AxialControlsWithSingleSide.args = {
 	sides: [ 'horizontal' ],
 	splitOnAxis: true,


```
</details>

Reply:
Great suggestion! I definitely was backtracking to try to match trunk but this makes more sense.